guides/common/attributes-satellite.adoc

```diff
@@ -9,6 +9,8 @@
 :PlanningDocURL: {BaseURL}planning_for_red_hat_satellite/index#
 :ProvisioningDocURL: {BaseURL}provisioning_guide/index#
 :UpgradingDocURL: {BaseURL}upgrading_and_updating_red_hat_satellite/index#
+// The following docs are in the downstream only
+:PuppetDocURL: {BaseURL}puppet_guide/index#
```

Comment:
From a first glance, I don't see any "new" guide. If so, you'd need the whole skeleton. See https://github.com/theforeman/foreman-documentation/pull/777 where I have a draft PR to create a new guide (new folder, new symlinks, etc.)

So, IMHO remove both lines.

Reply:
It's referring to this one https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/puppet_guide/index